.gitlab-ci.yml

```diff
@@ -327,15 +327,16 @@ fuzz-tests:
           done
         - if [ $all_tests_passed -eq 0 ]; then exit 0; fi
         - |
-             curl -X "POST" "https://api.github.com/repos/paritytech/ink/issues?state=all" \
+             curl -X "POST" "https://api.github.com/repos/paritytech/ink/issues" \
                 -H "Cookie: logged_in=no" \
                 -H "Authorization: token ${GITHUB_TOKEN}" \
-                -H "Content-Type: text/plain; charset=utf-8" \
+                -H "Content-Type: application/json; charset=utf-8" \
                 -d $'{
-                    "title": "[ci] Failing fuzz tests on master ('"$( date +"%d %b %Y" )"'),
-                    "body": "This CI job '"${CI_JOB_ID}"' just failed: '"${CI_JOB_URL}"'.\n\nThe offending commit is `['"${CI_COMMIT_TITLE}"'](https://github.com/paritytech/ink/commit/'"${CI_COMMIT_SHA}"')`.",
+                    "title": "[ci] Failing fuzz tests on master ('"$( date +"%d %b %Y" )"')",
+                    "body": "The CI job ['"${CI_JOB_ID}"']('"${CI_JOB_URL}"') just failed.\n\nThe offending commit is ['"${CI_COMMIT_TITLE}"'](https://github.com/paritytech/ink/commit/'"${CI_COMMIT_SHA}"').",
                     "assignees": [],
                     "labels": [
                         "P-high"
                     ]
                 }'
+        - exit 1
```

Comment:
so now we always fail it?

Reply:
Nope, in line 328 there is an early return if all tests succeeded:
```
- if [ $all_tests_passed -eq 0 ]; then exit 0; fi
```
The issue whiched is fixed with this `- exit 1`is that this `fuzz` CI stage always exited with `0`, since the last statement was `curl` and not the `cargo test …`. So the CI was green, despite of a fuzz test having failed.

Reply:
maybe then `exit "$all_tests_passed"`?

Reply:
That's a great suggestion! I just changed it.